### ISSUE TITLE
Some missing functions (eg new Date, JSON.stringify) in TypeScript

### ISSUE THREAD

**Issue by mcintyre94** [slipHQ/run-wasm#69]
**Describe the bug**

There seems to be some functions that are available in the browser/TS playground but aren't available in our run-wasm TS compiler. So far I've noticed it with `Date` and `JSON`. 

**To Reproduce**

1. Go to https://www.runwasm.com/ts
2. Type `console.log(new Date())` in the input
3. Click Run code

**Expected behavior**

Output should display a date eg. `Wed Sep 29 2021 19:03:21 GMT+0100 (British Summer Time)`

**Screenshots**

We get an error saying Date undefined:

![Screenshot 2021-09-29 at 19 04 12](https://user-images.githubusercontent.com/1711350/135324495-58e8bac3-4480-4c4d-8fe1-c586ea738402.png)

**Desktop (please complete the following information):**

- OS: macOS 11.6
- Browser Vivaldi
- Version 4.1.2369.21

**Additional context** Add any other context about the problem here.

- I've found this tutorial mentioning that some global functions are in Typescript libs: https://tutorial.tips/solved-cannot-find-name-do-you-need-to-change-your-target-library/
- You can see an example of the compiler options used in the NextJS example here: https://github.com/slipHQ/run-wasm/blob/main/example-nextjs/tsconfig.json#L2
- We currently pass empty options to our TS compiler: https://github.com/slipHQ/run-wasm/blob/main/src/RunWasmClient.ts#L96
- I've tried setting that `options` to match the example-nextjs one (I had to remove the isolatedModules line to stop it erroring):

```
  const options = {
    "target": "es5",
    "lib": ["dom", "dom.iterable", "esnext"],
    "allowJs": true,
    "skipLibCheck": true,
    "strict": false,
    "forceConsistentCasingInFileNames": true,
    "noEmit": true,
    "esModuleInterop": true,
    "module": "esnext",
    "moduleResolution": "node",
    "resolveJsonModule": true,
    "jsx": "preserve"
  }
```

But even with this I get the same error. Anyone with any more knowledge of the TypeScript compiler than me who can point in the right direction?

**Comment by kennethcassel** [comment]
Some great info in #68 on this for anyone onlooking

**Comment by kennethcassel** [comment]
Closed with #79
